src/status_im/ui/screens/wallet/views.cljs

```diff
@@ -28,7 +28,7 @@
     [react/view {:style styles/total-balance}
      [react/text {:style               styles/total-balance-value
                   :accessibility-label :total-amount-value-text}
-      value]
+      (i18n/format-currency (:code currency) value)]
```

Comment:
I'd suggest keeping views as simple as possible and move computation to subscription https://github.com/Day8/re-frame/blob/master/docs/SubscriptionsCleanup.md

Reply:
@dmitryn Got it, thanks, will action! 👍

Reply:
@dmitryn Done :)  :muscle: